lib/breathe/client.rb

```diff
@@ -38,6 +39,7 @@ def agent
       Sawyer::Agent.new(BASE_URL, links_parser: Sawyer::LinkParsers::Simple.new) do |http|
         http.headers["Content-Type"] = "application/json"
         http.headers["X-Api-Key"] = api_key
+        http.headers["Authorization"] = "Bearer " + api_key
```

Comment:
We should update the redacted client to use the `X-Api-Key` header rather than make the change here